Cubical/Structures/Ring.agda

```diff
@@ -23,9 +23,12 @@ raw-ring-structure X = (X → X → X) × X × (X → X → X)
 
 -- Maybe this is not the best way? (Suggestions welcome, maybe having raw-monoid-iso defined?)
 raw-ring-is-SNS : SNS {ℓ} raw-ring-structure _
-raw-ring-is-SNS = join-SNS (nAryFunIso 2) (nAryFunSNS 2)
-                           (join-iso pointed-iso (nAryFunIso 2))
-                           (join-SNS pointed-iso pointed-is-SNS (nAryFunIso 2) (nAryFunSNS 2))
+raw-ring-is-SNS = join-SNS (binaryFunIso pointed-iso)
```

Comment:
Maybe `binaryFunIso pointed-iso` should be stated as a lemma as you use it multiple times? Same for some other arguments here...